pkg/graph/resolvers/mto_milestone.go

```diff
@@ -101,6 +101,10 @@ func MTOMilestoneUpdate(ctx context.Context, logger *zap.Logger, principal authe
 	if err != nil {
 		return nil, fmt.Errorf("unable to update MTO Milestone. Err %w", err)
 	}
+	// Clear out the name returned from the database for convenience so we don't try to set the custom name field
+	if existing.AddedFromMilestoneLibrary() {
+		existing.Name = nil
```

Comment:
Maybe throw a `logger.Warn()` in here -- we shouldn't expect this to happen, but might be nice to find it if it does :)

Reply:
Thanks @ClayBenson94! We actually do expect this to happen every time there is a milestone sourced from a common milestone being updated, so I don't think we want to log a warning. (Because we return the common milestones name from SQL). We got around this previously because there were two distinct fields for operational needs.